### PR TITLE
Add UUIDs command

### PR DESCRIPTION
**Pull request by ripleyaffect**

### CODE REVIEW

File: src/lib/getLibraryFiles.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  // Create the pattern for finding all content files
+  const globPattern = `${targetDirectory}/**/content.md`;
+
+  // Get all file paths matching the pattern
+  const filePaths = await globby(globPattern, null);
+
+  const fileObjects = [];
+
+  // Create lightweight file objects with path and data
+  for (const filePath of filePaths) {
```
Comment (learykara) [inline src/lib/getLibraryFiles.js]: this works? I thought something like this had to be a `let`

File: src/lib/getLibraryFiles.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const fs = require('fs-extra');
+const globby = require('globby');
+
+const getLibraryFile = async filePath => {
+  try {
+    const fileRaw = await fs.readFile(filePath, 'utf8');
+
+    return {
+      path: filePath,
+      data: frontMatter(fileRaw),
```
Comment (learykara) [inline src/lib/getLibraryFiles.js]: not alpha 😱

File: src/lib/createUuids.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  // Add missing uuid
+  if (!data.uuid) {
+    data.uuid = uuidv4();
+  }
+};
+
+module.exports = async (files, options) => {
+  // Process the file objects
+  for (const file of files) {
+    await processFile(file, options);
```
Comment (learykara) [inline src/lib/createUuids.js]: why does this one need to be async?

File: src/commands/uuids.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+const glob = require('glob');
+
+const { createUuids, getLibraryFiles, setLibraryFiles } = require('../lib');
+
+class UuidsCommand extends Command {
+  async run() {
+    const { flags } = this.parse(UuidsCommand);
+
+    const directory = flags.directory;
+    const strict = flags.strict;
```
Comment (learykara) [inline src/commands/uuids.js]: `const { directory, strict } = flags;`?

File: src/commands/uuids.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    } catch (error) {
+      this.error(error);
+    }
+  }
+}
+
+UuidsCommand.description = `Generate uuids for library
+Each directory in \`library\` should have a \`content.md\` file with metadata
+at the top. This function looks at all the content and adds a new uuid to the
+metadata if it's not already present.
```
Comment (learykara) [inline src/commands/uuids.js]: maybe worth adding a line about how to run this and set flags via command line?

Comment (ripleyaffect) [reply src/commands/uuids.js]: all that is generated by `oclif`

Comment (learykara) [reply src/commands/uuids.js]: oh neat

**Pull request merged**